.github/workflows/tf_apply_staging.yml

```diff
@@ -60,3 +60,7 @@ jobs:
       - name: Apply cloudfront 
         working-directory: terragrunt/env/staging/cloudfront
         run: terragrunt apply --terragrunt-non-interactive -auto-approve
+
+      - name: Apply dynamodb 
+        working-directory: terragrunt/env/staging/dynamodb
+        run: terragrunt apply --terragrunt-non-interactive -auto-approve
```

Comment:
Same comment about before the `api` module step.

Reply:
Order has been changed.